src/Uno.UI/UI/Xaml/Style/Generic/SystemResources.xaml

```diff
@@ -1378,8 +1378,8 @@
 			<x:Double x:Key="ComboBoxThemeMinWidth">64</x:Double>
 			<x:Double x:Key="ComboBoxPopupThemeMinWidth">80</x:Double>
 			<x:Double x:Key="ComboBoxPopupThemeTouchMinWidth">240</x:Double>
-			<x:Double x:Key="ControlContentThemeFontSize">15</x:Double>
-			<x:Double x:Key="ContentControlFontSize">15</x:Double>
+			<x:Double x:Key="ControlContentThemeFontSize">14</x:Double>
```

Comment:
Is this change based on the UWP/WinUI sources?

Reply:
Yes.